### PR TITLE
feat(tool-nuxt): add starter

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR adds a nuxt starter for tool plugin.

Currently this PR has duplicated `space-plugins/nuxt-starter` into `tool-plugins/nuxt-starter`. The real changes are made in a separate PR for the sake of reviewability (https://github.com/storyblok/space-tool-plugins/pull/78).

## Why?

JIRA: SHAPE-6288

### CODE REVIEW

File: tool-plugins/nuxt-starter/app.config.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default defineAppConfig({
+	type: 'space-plugin',
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/app.config.ts:2]: this is tool-plugin right?

File: tool-plugins/nuxt-starter/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+## Tutorial Video
+
+If you prefer video tutorials over text, this is for you:
+
+https://www.loom.com/share/f56defc5bfcf4fa8bce8682386f8352b?sid=8820f1b2-4229-4b05-9f22-4855e6dfccc5
+
+## Getting Started
+
+```sh
+npx giget@latest gh:storyblok/space-tool-plugins/space-plugins/nuxt-starter YOUR-PROJECT-NAME
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/README.md:14]:
```suggestion
npx giget@latest gh:storyblok/space-tool-plugins/tool-plugins/nuxt-starter YOUR-PROJECT-NAME
```

File: tool-plugins/nuxt-starter/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+# `space-plugin-nuxt-starter`
+
+This Nuxt starter is a Storyblok Space Plugin app that includes the basic authentication flow.
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/README.md:3]:
```suggestion
This Nuxt starter is a Storyblok Tool Plugin app that includes the basic authentication flow.
```

File: tool-plugins/nuxt-starter/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+5. App Bridge is an extra authentication layer recently introduced for Space Plugins and Tool Plugins. This starter assumes you've enabled App Bridge on the Settings page. Documentation on App Bridge will come in the near future, but you don't need to know about its inner process. This starter takes care of it out-of-the-box.
+
+<img src="./docs/app-bridge.png" alt="App Bridge" width="600" />
+
+If you don't see the UI to enable App Bridge, please reach out to [the Storyblok support](https://www.storyblok.com/trust-center/service-support). Or, you can disable App Bridge, by replacing your `app.config.ts` with the following.
+
+```ts
+export default defineAppConfig({
+	appBridge: {
+		type: 'space-plugin',
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/README.md:71]:
```suggestion
		type: 'tool-plugin',
```

File: tool-plugins/nuxt-starter/README.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+# `space-plugin-nuxt-starter`
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/README.md:1]:
```suggestion
# `tool-plugin-nuxt-starter`
```

File: tool-plugins/nuxt-starter/nuxt.config.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+export default defineNuxtConfig({
+	devtools: { enabled: true },
+	ssr: false,
+	extends: [
+		[
+			'github:storyblok/space-tool-plugins/space-plugins/nuxt-base',
+			{ install: true },
+		],
+		[
+			'github:storyblok/space-tool-plugins/space-plugins/nuxt-base-ui',
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/nuxt.config.ts:11]:
```suggestion
			'github:storyblok/space-tool-plugins/tool-plugins/nuxt-base-ui',
```

File: tool-plugins/nuxt-starter/nuxt.config.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+	devtools: { enabled: true },
+	ssr: false,
+	extends: [
+		[
+			'github:storyblok/space-tool-plugins/space-plugins/nuxt-base',
```
Comment (BibiSebi) [inline tool-plugins/nuxt-starter/nuxt.config.ts:7]:
```suggestion
			'github:storyblok/space-tool-plugins/tool-plugins/nuxt-base',
```

File: tool-plugins/nuxt-starter/app.config.ts (hunk 1 above)
Comment (eunjae-lee) [reply tool-plugins/nuxt-starter/app.config.ts:2]: Sorry I wasn't clear! #77 and #78 do the actual jobs, so you can review them. They fix all the things in this PR.

File: tool-plugins/nuxt-starter/nuxt.config.ts (hunk 6 above)
Comment (eunjae-lee) [reply tool-plugins/nuxt-starter/nuxt.config.ts:11]: I actually used the same nuxt-base and nuxt-base-ui from space-plugins folder. I didn't duplicate the same inside tool plugins folder.

File: tool-plugins/nuxt-starter/app.config.ts (hunk 1 above)
Comment (BibiSebi) [reply tool-plugins/nuxt-starter/app.config.ts:2]: Oh sorry, I misunderstood. I reviewed both already so I can also approve this one.

**Review by BibiSebi — APPROVED** [review]
Let's gooooo 🚀

**Pull request merged**